gulp/ProjectSpecs/DefaultSpecs.js

```diff
@@ -1,3 +1,24 @@
+/* 
+* Global constants info
+**/
+const constants = {
+    envType: {
+        development: 'dev', 
+        production: 'prod'
+    },
+    // list of files to be excluded from a specific platform
+    excludeFromTsTranspile: {
+        O11: [
+            './src/scripts/OutSystems/OSUI/Utils/PreviewInDevices/**/*'
+        ]
+    },
+    // list of platforms to compile and create scss files.
+    platforms: {
```

Comment:
target?

Reply:
To follow what we have it should also be in singular instead of plural

Reply:
Done!